src/pkg/backup/details/groups.go

```diff
@@ -42,8 +42,11 @@ type GroupsInfo struct {
 	Modified time.Time `json:"modified,omitempty"`
 
 	// Channels Specific
-	Message   ChannelMessageInfo `json:"message"`
-	LastReply ChannelMessageInfo `json:"lastReply"`
+	Message   ChannelMessageInfo `json:"message,omitempty"`
+	LastReply ChannelMessageInfo `json:"lastReply,omitempty"`
+
+	// Conversations Specific
+	Post ConversationPostInfo `json:"post,omitempty"`
```

Comment:
1) Can't we just reuse `ChannelMessageInfo` (maybe rename it)
1) Do we even need a separate field? Can we just fold into `Message` with `ItemType` deciding the type?

Reply:
I don't see the value in it.  A little bit of duplication here won't hurt us, and it'll help minimize confusion for SDK consumers.  Per itemType, that's a largely internal reference.  End consumers don't have a good way of knowing which available value do and do not get populated based on type, nor are we providing strict guarantees of stability there.